spk/nzbhydra/src/dsm-control.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+
+# Package
+PACKAGE="nzbhydra"
+DNAME="NZBHydra"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python"
+GIT_DIR="/usr/local/git"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${GIT_DIR}/bin:${PATH}"
+USER="nzbhydra"
+PYTHON="${INSTALL_DIR}/env/bin/python"
+NZBHYDRA="${INSTALL_DIR}/share/nzbhydra.py"
+PID_FILE="${INSTALL_DIR}/var/nzbhydra.pid"
+LOG_FILE="${INSTALL_DIR}/var/nzbhydra.log"
+DB_FILE="${INSTALL_DIR}/var/nzbhydra.db"
+CONF_FILE="${INSTALL_DIR}/var/settings.cfg"
+
+
+start_daemon ()
+{
+    su - ${USER} -c "PATH=${PATH} ${PYTHON} ${NZBHYDRA} --daemon --nobrowser --database ${DB_FILE} --config ${CONF_FILE} --logfile ${LOG_FILE} --pidfile ${PID_FILE} "
+    sleep 1
+    ps w | grep [${INSTALL_DIR}]/share/nzbhydra.py | awk '{print $1}' > ${PID_FILE}
```

Comment:
If nzbhydra is able to write a PIDFILE (via `--pidfile ${PID_FILE}`), this would seem unnecessary. Please verify.